Reference/Searching/Examine/pdfindex-multisearcher.md

```diff
@@ -1,78 +1,51 @@
----
-versionFrom: 8.0.0
-versionTo: 8.0.0
+﻿---
+versionFrom: 9.0.0
+versionTo: 10.0.0
 ---
 
 # PDF indexes and multisearchers
 
-:::note
-This document has been verified for Umbraco 8.
-
-If you are using Umbraco 9 or later versions, please refer to the note on the [Examine documentation landing page](index.md) for more details.
-:::
-
 If you want to index PDF files and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
 
 ## Installation
 
 Install with Nuget:
-`Install-Package UmbracoCms.UmbracoExamine.PDF`
-
-You will then have a new Examine index called "PDFIndex" available.
+`dotnet add package Umbraco.ExaminePDF`
 
-## Multi index searchers
+This will create a new Examine index called "PDFIndex", which will appear in "Examine Management" dashboard under the "Settings" section. Using this index you can start searching the contents of any PDF files uploaded to the media section.
 
-To use the multisearcher in Umbraco 8, you can instantiate it when needed like:
+![image](https://user-images.githubusercontent.com/7405322/189886089-d23b45c7-814b-4101-b143-31c5cd9fa655.png)
 
-```csharp
-using(var multiSearcher = new MultiIndexSearcher("MultiSearcher", new IIndex[] {
-    externalIndex,
-    pdfIndex
-}))
-{
- ...
-};
-```
 
-Or you can register a multi index searcher with the ExamineManager on startup like:
+## Multi index searchers
+A multi index searcher, is a searcher, than can search multiple indexes, this can be really helpful when you for example want to search both the external and internal indexes.
```

Comment:
```suggestion
A multi-index searcher is a searcher that can search multiple indexes. This can be helpful when you for example want to search both the external and internal indexes.
```